lusidtools/apps/flush_transactions.py

```diff
@@ -0,0 +1,174 @@
+        if curr_count + str_len >= remaining_count:
+            batches.append(curr_batch)
+            curr_batch = []
+            curr_count = 0
+
+        curr_batch.append(txn_id)
+        curr_count = curr_count + str_len
+
+    # Create remainder final batch if any remainder exists
+    if curr_batch:
+        batches.append(curr_batch)
+
+    return batches
+
+
+def get_ids_from_txns(txns: lusid.VersionedResourceListOfTransaction):
+    """
+        Takes a given input list of transactions and extracts the transaction IDs from them
+
+        Parameters
+        ----------
+        txns : lusid.VersionedResourceListOfTransaction
+            Unedited list of transactions from GetTransactions response
+
+        Returns
+        -------
+        txnIDs : list
+            Returns a list of the transaction IDs for the observed transactions
+    """
+    return [txn.transaction_id for txn in txns.values]
+
+
+def flush(scope, code, fromDate, toDate, api_factory):
+    """
+            Cancels all transactions found in a given date range for a specific portfolio
+
+            Parameters
+            ----------
+            scope : str
+                Scope of the portfolio in question
+            code : str
+                Code of the portfolio in question
+            fromDate : str
+                Date after which transactions will be retrieved
+            toDate : str
+                Date up to which transactions will be retrieved
+        """
+    # Initialise the api
+    secrets_path = str(
+        pathlib.Path(__file__).parent.parent.parent.joinpath("tests", "secrets.json")
+    )
+
+    transaction_portfolios_api = api_factory.build(lusid.api.TransactionPortfoliosApi)
+
+    # Get the Transaction Data
+    print("Getting Transaction Data")
+    transactions = transaction_portfolios_api.get_transactions(
+        scope,
+        code,
+        from_transaction_date=fromDate,
+        to_transaction_date=toDate,
+        limit=5000,
+    )
+
+    txn_ids = get_ids_from_txns(transactions)
+
+    # Manage pagination of GetTransactions
+    next_transaction_page = transactions.next_page
+    while next_transaction_page is not None:
+        page_of_transactions = transaction_portfolios_api.get_transactions(
+            scope,
+            code,
+            from_transaction_date=fromDate,
+            to_transaction_date=toDate,
+            limit=5000,
+            page=next_transaction_page,
+        )
+
+        txn_ids = txn_ids + get_ids_from_txns(page_of_transactions)
+
+        next_transaction_page = page_of_transactions.next_page
+
+    if not txn_ids:
+        print("No transactions found between the specified dates")
+        logging.error("No transactions found between the specified dates")
+        return
+
+    logging.info(f"Found {len(txn_ids)} transactions between the specified dates")
+
+    # Batch the transactions
+    character_count = 4000
+    txn_ids = transaction_batcher_by_character_count(
```

Comment:
Can you rename this variable? Something like `txn_id_batches`  or `txn_batches` would make it easier to understand